### PR TITLE
Update to Emoji v13 (2020)

### PR DESCRIPTION
**Pull request by pajlada**
The plan is to:
 - [ ] Update the emoji stuff to support Unicode 12.1 emojis and their new tone stuff
 - [ ] Upload emojis to a proper CDN instead of using a random server

### CODE REVIEW

**Comment by RAnders00** [comment]
https://www.jsdelivr.com/package/npm/emoji-datasource-twitter?path=img%2Ftwitter%2F64 twitter emojis are on jsdelivr too as far as i can see

**Comment by RAnders00** [comment]
An issue I've found: Some modifiers are not rendered as images when there are images for them, for example 🦲 (EMOJI COMPONENT BALD) renders as https://abs-0.twimg.com/emoji/v2/svg/1f9b2.svg when used on twitter.com (used as a reference for their Twitter set of emoji images) however in chatterino it does not render: 
image

Emojipedia also lists the twitter image for the "bald" modifier alone: https://emojipedia.org/bald/
image

**Comment by zneix** [comment]
Fixing this would close #869

**Comment by alazymeme** [comment]
Hello any news on this pr? I'm patiently waiting to use the pinching hand emoji in chatterino 🤏

**Comment by fourtf** [comment]
Is this still being worked on or can I close?

**Comment by pajlada** [comment]
Leave this up

**Comment by alazymeme** [comment]
Looking forward to using the new emojis in chatterino 😁

**Comment by alazymeme** [comment]
New year, new emojis 😄 can't wait for this to be implemented!

**Comment by alazymeme** [comment]
Can't wait to prove these nerds wrong when I can see the window emoji 😲

**Comment by Mm2PL** [comment]
Can't wait for this to be merged

File: src/providers/emoji/Emojis.cpp
Comment (zneix) [inline src/providers/emoji/Emojis.cpp:392]: We should not use `printf` here

**Review by zneix — CHANGES_REQUESTED** [review]
Looks good to me, new emojis work fine. Worth noting there's a very slow emoji load at the first launch, but since it seems to be cached and loads quicker later I guess it's fine :+1:

**Comment by pajlada** [comment]
> Looks good to me, new emojis work fine. Worth noting there's a very slow emoji load at the first launch, but since it seems to be cached and loads quicker later I guess it's fine +1

Thanks for the review!
The slow speed is at least just as slow as it's been before, not slower, as it's hosted on the same provider & server and the filesizes are very similar for these set of emojis.

Still interested in finding a "true CDN" that isn't slow af, but the b2+cloudflare combo was megaslow

**Comment by alazymeme** [comment]
We stopped using JoyPixels didn't we? Was looking at a basic CDN like this: https://cdnjs.com/libraries/emoji-toolkit

**Comment by pajlada** [comment]
> We stopped using JoyPixels didn't we? Was looking at a basic CDN like this: https://cdnjs.com/libraries/emoji-toolkit

We did, yes

**Comment by pajlada** [comment]
Waiting for #2453 to be merged in before running final tests on this and merging it

**Pull request merged**
